src/tutorials/boilerplates/boilerplate-code.js

```diff
@@ -31,8 +31,8 @@ return run
 `
 
 // Optional: Use the modules option if you need extra
-// modules available in the solution code
-const modules = { cids: require('cids') }
+// modules available in the validation code
```

Comment:
@terichadbourne  Should be solution code, not validation. Basically modules available to the user when writing up the solution code.

```suggestion
// modules available in the solution code
```